Bio/Affy/CelFile.py

```diff
@@ -185,7 +318,3 @@ def read(handle):
         else:
             continue
     return record
-
-if __name__ == "__main__":
-    from Bio._utils import run_doctest
-    run_doctest()
```

Comment:
I've put it back in, but I can't get it to work neither under python2 nor python3 due to import issues. (python2 tries to import a version of biopython I installed through pip, and python3 can't import Bio).

The docstrings work through CI though.